cli/chain.go

```diff
@@ -399,6 +399,10 @@ var ChainSetHeadCmd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
+			return IncorrectNumArgs(cctx)
+		}
+
```

Comment:
This one can be removed, see the block just below, this command has a subtle usage - you can either set it to `--genesis`, set an `--epoch X`, _or_ provide a tipset.
Not a very graceful command, we should probably make it more standard, maybe a `--tipset` instead of using the arg. But for now this can be removed and we'll leave it alone.